tiny-composer-installer.php

```diff
@@ -60,7 +61,7 @@
 $phar = fetch_phar($base_url, $version);
 $signature = fetch_signature($base_url, $version);
 check_signature($phar, $signature, $signature_key);
-echo write_phar($phar) . PHP_EOL;
+echo write_phar($phar, $default_filename) . PHP_EOL;
```

Comment:
Just a suggestion (not the php way, I know, feel free to oppose), but I think creating a main function (and immediately calling it) with all the code which is not in functions currently would greatly improve readability.

Reply:
Maybe, yeah. But it’s not something for this PR. So, WONTFIX here but I’ll think about it.

Reply:
👍